packages/lodestar/src/sync/initial/fast/index.ts

```diff
@@ -62,20 +69,25 @@ export class FastSync extends (EventEmitter as {new (): InitialSyncEventEmitter}
     this.chain.on("processedCheckpoint", this.checkSyncCompleted);
     this.chain.on("processedBlock", this.checkSyncProgress);
     this.syncTriggerSource = pushable<ISlotRange>();
-    this.blockImportTarget = this.chain.forkChoice.headBlockSlot();
+    const headSlot = this.chain.forkChoice.headBlockSlot();
```

Comment:
why not fetch finalized slot block from db on start? It would ignore all unfinalized blocks